src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -230,11 +230,17 @@ def create_filesystem(self, fs_name, metadata_pool, data_pool):
                    'data': data_pool}
         return self.mgr.mon_command(command)
 
-    def remove_filesystem(self, fs_name):
+    def remove_filesystem(self, fs_name, confirm):
+        if confirm != "--yes-i-really-mean-it":
+            return -errno.EPERM, "", "this is a DESTRUCTIVE operation that will make " \
+                "data in your filesystem permanently inaccessible. " \
+                "Add --yes-i-really-mean-it if you are sure you wish to continue."
```

Comment:
I don't find this instruction clear. Add '--yes-i-really-meant' where? The user has to re-issue the command and append the flag, right?

e.g.,

See the stderr message when you try to delete a RADOS pool.
```
$ ceph osd pool delete cephfs.a.data
Error EPERM: WARNING: this will *PERMANENTLY DESTROY* all data stored in pool cephfs.a.data.  If you are *ABSOLUTELY CERTAIN* that is what you want, pass the pool name *twice*, followed by --yes-i-really-really-mean-it.
```

So maybe something like,
```
Error EPERM: WARNING: this will *PERMANENTLY DESTROY* all data stored in the filesystem <fs-name>.  If you are *ABSOLUTELY CERTAIN* that is what you want,  re-issue the command followed by --yes-i-really-mean-it.
```